modules/get-started/pages/licensing/overview.adoc

```diff
@@ -31,27 +31,74 @@ The Enterprise Edition is licensed with the https://github.com/redpanda-data/red
 
 [[self-managed]]
 === Enterprise features in Redpanda Self-Managed
 
-The following enterprise features for Redpanda Self-Managed are activated with a valid Enterprise Edition license key:
+The following table lists the available Enterprise features for Redpanda Self-Managed, along with the sanctions applied if the Enterprise license is invalid or has expired.
+
+[cols="1,2,2"]
+|===
+| Feature | Description | Sanction if License Invalid
+
+| xref:manage:tiered-storage.adoc[Tiered Storage]
+| Enables storage of data in cloud object storage for long-term retention and retrieval.
+| Topics cannot be created or modified to enable Tiered Storage features. Additional partitions cannot be added to topics that have any Tiered Storage properties enabled.
+
+| xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Continuous Data Balancing]
+| Automatically balances partitions across a cluster to optimize resource use and performance.
+| Continuous balancing is disabled, reverting to the default `node_add` setting that balances partitions only after a broker is added to the cluster.
+
+| xref:manage:cluster-maintenance/cluster-balancing.adoc#intra-broker-partition-balancing[Intra-Broker Partition Balancing (`core_balancing_continuous`)]
+| Balances partition replicas across CPU cores in an individual broker to optimize disk space usage.
+| Intra-broker balancing is disabled.
+
+| xref:manage:remote-read-replicas.adoc[Remote Read Replicas]
+| Enables remote clusters to read data stored in object storage for disaster recovery.
+| Remote Read Replica topics cannot be created or modified.
 
-include::get-started:partial$licensing/enterprise-features.adoc[tag=self-managed]
+| xref:manage:audit-logging.adoc[Audit Logging]
+| Records detailed logs of cluster activities for compliance and monitoring.
+| Read access to the audit log topic is denied, but logging continues.
+
+| xref:manage:security/fips-compliance.adoc[FIPS Compliance]
+| Enables compliance with FIPS security standards for encryption.
+| None
+
+| xref:manage:schema-reg/schema-id-validation.adoc[Server-Side Schema ID Validation]
+| Validates schema IDs server-side to ensure schema compatibility. With schema ID validation, records associated with unregistered schemas are detected and dropped by a broker rather than a consumer.
+| Topics with schema validation settings cannot be created or modified.
+
+| xref:manage:security/authentication.adoc#kerberos[Kerberos authentication]
+| Provides secure Kerberos-based authentication.
+| None
+
+| xref:manage:security/authentication.adoc#oidc[OAUTHBEARER/OIDC authentication]
+| Allows for OAUTHBEARER and OpenID Connect (OIDC) authentication.
+| None
+
+| xref:manage:security/authorization/rbac.adoc[Role-Based Access Control (RBAC)]
+| Manages user roles and permissions within the cluster.
+| Roles and ACLs cannot be created or modified. Role deletion is allowed.
```

Comment:
suggestion: Specifically, ACLs associated with roles cannot be created.  I feel like this language suggests overall ACL creation is disallowed.